proxies/live/apiproxy/policies/AssignMessage.Errors.CatchAllMessage.xml

```diff
@@ -1,27 +1,33 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.Errors.CatchAllMessage">
-  <Set>
-    <Payload contentType="application/json" variablePrefix="%" variableSuffix="#">
-    {
+    <Set>
+      <Payload contentType="application/json" variablePrefix="%" variableSuffix="#">
+      {
         "resourceType": "OperationOutcome",
+        "meta": {
+          "profile": [
+            "https://fhir.hl7.org.uk/StructureDefinition/NHSDigital-OperationOutcome"
+          ]
+        },
         "issue": [
-            {
+          {
             "severity": "error",
-            "code": "%error.status.code#",
-            "details": {
-                "coding": [
-                {
-                    "code": "%error.status.code#",
-                    "display": "%error.reason.phrase#"
-                }
-                ]
+            "code": "%erspca.error.code#",
```

Comment:
What if something goes wrong in an APIM shared flow, I guess this variable won't be set in this case? I wonder if we should be using `error.*` here instead and setting that variable in our fault rules to allow this policy to generically apply to both eRS and APIM errors.